src/event.rs

```diff
@@ -47,7 +47,7 @@ impl Event {
     /// spending plan is valid.
     pub fn verify(&self) -> bool {
         match *self {
-            Event::Transaction(ref tr) => tr.verify(),
+            Event::Transaction(ref tr) => tr.plan_verify(),
```

Comment:
You should not have needed to make this change. I took verify() out of skel's codepath.